packages/zowe-explorer/__tests__/__integration__/bdd/features/Settings.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Settings Actions
+
+Scenario: User can locate Zowe Explorer settings
```

Comment:
`Given` steps are needed when setup is involved for a test scenario. In this case, no set up is needed to perform the action, so I've left it out here 👍 

I can add a doc to the wiki linking to best practices for Cucumber tests, in case we want a reference moving forward!

Reply:
I think `Given` is mostly used for the setup work that happens before the actual use case
`When` is for the actual scenario being tested
`Then` is mostly for the results
and `And` is just to add to either of the above


<details><summary>This is just a dummy example 😋 </summary>


```
Given that Zowe Explorer is installed
And that a user clicks on the Zowe Explorer icon
And that the Datasets Tree view is loaded
When the user right clicks on the default profile
And selects `Manage Profile` 
Then a quick-pick opens 
And contains the labels [x, y, z]
```
</details>